ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+
+      ccloud kafka topic consume demo-topic-1 -b
+
+#. Verify you see the following output:
+
+   .. code-block:: text
+
+      Starting Kafka Consumer. ^C or ^D to exit
+      2
+      3
+      9
+      4
+      5
+      7
+      10
+      1
+      6
+      8
+
+
+Create a service account key/secret pair
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the following commmand to create a new service account:
+
+   .. code-block:: bash
+
+      ccloud service-account create demo-app-3288 --description demo-app-3288 -o json
+
+
+   .. code-block:: text
+
+      {
+         "id": 104349,
+         "name": "demo-app-3288",
+          "description": "demo-app-3288"
+      }
+
+#. Create an API key and secret for the service account ``104349`` by running
+   the following account:
+
+   .. code-block:: bash
+
+      ccloud api-key create --service-account 104349 --resource lkc-x6m01 -o json
+
+   .. code-block:: text
+
+      {
+        "key": "ESN5FSNDHOFFSUEV",
+        "secret": "nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB"
+      }
+
+#. Complete the following steps to create a local configuration file
+   ``/tmp/client.config`` with |ccloud| connection information using the newly
+   created API key and secret:
+
+   a. Open the config file:
+
+      cat /tmp/client.config
+
+   b. Update  ``/tmp/client.config``  with the following parameters:
+
+      .. code-block:: text
+
+         ssl.endpoint.identification.algorithm=https
+         sasl.mechanism=PLAIN
+         security.protocol=SASL_SSL
+         bootstrap.servers=pkc-4kgmg.us-west-2.aws.confluent.cloud:9092
+         sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="ESN5FSNDHOFFSUEV" password\="nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB";
+
+   c. Wait 90 seconds for the service account credentials to propagate
+
+   d. By default, no ACLs are configured. To confirm, run the following command:
+
+      .. code-block:: bash
+
+         ccloud kafka acl list --service-account 104349
+
+      You should see the following output:
+
+      .. code-block:: text
+
+            ServiceAccountId | Permission | Operation | Resource | Name | Type
+          +------------------+------------+-----------+----------+------+------+
+
+
+Run a Java producer before and after configuring ACLs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Run the Java producer to ``demo-topic-1`` before configuring ACLs (expected to fail):
+
+   .. code-block:: bash
+
+      mvn -q -f ../../clients/cloud/java/pom.xml exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.ProducerExample" -Dexec.args="/tmp/client.config demo-topic-1" -Dlog4j.configuration=file:log4j.properties > /tmp/log.1 2>&1
+
+#. Verify you see ``org.apache.kafka.common.errors.TopicAuthorizationException``
+   in the logs as shown in the following example (expected because there are no ACLs to allow this client application):
```

Comment:
```suggestion
   in the log file ``/tmp/log.1`` as shown in the following example (expected because there are no ACLs to allow this client application):
```

Reply:
updated in new PR